docs/source/conf.py

```diff
@@ -17,6 +17,9 @@
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
 sys.path.append(os.path.abspath('sphinxext'))
+sys.path.append(os.path.abspath(os.path.join('..','..')))
+
+from chaco import __version__
```

Comment:
Or you could just pull the version number from `_version.py` without importing it, like [you did for traitsui](https://github.com/enthought/traitsui/blob/master/docs/source/conf.py#L45-L57).